### PR TITLE
Mlair/46 swift UI refactor for cloud functions

### LINKED ISSUE #46: Swift: UI refactor for cloud functions
continue refactor for cloud functions operations

- [x] edit user
- [x] edit club
- [x] tests

### PR DESCRIPTION
**Pull request by m-lair**
closes #46

### CODE REVIEW

File: MovieClub/Models/Club.swift [hunk 1]
```diff
@@ -110,7 +117,7 @@ final class MovieClub: Identifiable, Decodable, Hashable, Equatable {
         case movieEndDate
         case ownerId
         case isPublic
-        case banner
+        case banner = "image"
```
Comment (Codball) [inline MovieClub/Models/Club.swift:120]: maybe we should align on something like `bannerImage` instead?

File: MovieClub/Views/Clubs/ClubEditView.swift [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            return
+        }
+        guard !name.isEmpty else {
+            errorShowing = true
+            errorMessage = "Name cannot be empty"
+            return
+        }
+        guard !description.isEmpty else {
+            errorShowing = true
+            errorMessage = "Name cannot be empty"
```
Comment (Codball) [inline MovieClub/Views/Clubs/ClubEditView.swift:93]:
```suggestion
            errorMessage = "description cannot be empty"
```

File: MovieClub/Views/Clubs/ClubEditView.swift [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+            return
+        }
+        guard timeInterval != 0 else {
+            errorShowing = true
+            errorMessage = "Name cannot be empty"
```
Comment (Codball) [inline MovieClub/Views/Clubs/ClubEditView.swift:98]:
```suggestion
            errorMessage = "Time Interval cannot be empty"
```

File: MovieClub/Models/Club.swift (hunk 1 above)
Comment (m-lair) [reply MovieClub/Models/Club.swift:120]: was thinking the same with user image as well. 

maybe just club/user image?

**Review by Codball — APPROVED** [review]
couple small comments but lgtm 👍

File: MovieClub/Models/Club.swift (hunk 1 above)
Comment (Codball) [reply MovieClub/Models/Club.swift:120]: works for me

**Pull request merged**